docs/visual-testing/integrations/csharp.md

```diff
@@ -0,0 +1,319 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# C#/.Net WebDriver Integration
+
+:::note Important
+Access to this feature is currently limited to Enterprise customers as part of our commitment to providing tailored solutions. We are excited to announce that self-service access is under development and will be released shortly. Stay tuned!
+:::
+
+## Introduction
+
+This guide requires an existing C# NUnit / xUnit setup.<br />
+You can alternatively take a look to our [example repository](#examples).
+
+Sauce Visual provides an library allowing integration with [WebDriver](https://www.selenium.dev/documentation/webdriver/).
+
+Sauce Visual plugin provides a library exposing a `VisualClient` object that provides actions:
+
+- `visualClient.VisualCheck()`: Takes a screenshot and send it to Sauce Visual for comparison.
+- `visualClient.VisualResults()`: Waits for all diff calculations to complete and returns a summary of results.
+  See [Test results summary](#test-results-summary) for more details about summary format and sample usage.
+
+## Quickstart
+
+### Step 1: Add Sauce Visual dependecy
+
+Add [Sauce Visual](https://www.nuget.org/packages/SauceLabs.Visual/) dependency to your `.csproj`
+```xml
+  <!-- FIXME USE Example Repo Reference -->
+  <ItemGroup>
+    ...
+    <PackageReference Include="SauceLabs.Visual" Version="0.0.1" />
+    ...
+  </ItemGroup>
+```
+
+_Note: You can find the latest versions available [here](https://www.nuget.org/packages/SauceLabs.Visual#readme-body-tab)._
+
+### Step 2: Configure Visual Testing integration
+
+Declare a RemoteWebDriver and a VisualClient instance as class properties
+
+```csharp
+using OpenQA.Selenium.Remote;
+using SauceLabs.Visual;
+
+private RemoteWebDriver Driver { get; set; }
+private VisualClient VisualClient { get; set; }
+```
+
+Initialize `RemoteWebDriver` and `VisualClient`
+
+<Tabs
+defaultValue="NUnit"
+  values={[
+    {label: 'NUnit', value: 'NUnit'},
+    {label: 'xUnit', value: 'xUnit'},
+  ]}>
+<TabItem value="NUnit">
+
+```csharp
+    [OneTimeSetUp]
+    public void Setup()
+    {
+      var sauceUsername = "YOUR_USERNAME";
+      var sauceAccessKey = "YOUR_ACCESS_KEY";
+      var sauceUrl = "https://ondemand.us-west-1.saucelabs.com:443/wd/hub";
+
+      var browserOptions = new ChromeOptions();
+      browserOptions.PlatformName = "Windows 11";
+      browserOptions.BrowserVersion = "latest";
+
+      var sauceOptions = new Dictionary<string, object>();
+      sauceOptions.Add("username", sauceUsername);
+      sauceOptions.Add("accessKey", sauceAccessKey);
+      browserOptions.AddAdditionalOption("sauce:options", sauceOptions);
+
+      Driver = new RemoteWebDriver(sauceUrl, browserOptions);
+
+      VisualClient = new VisualClient(Driver, Region.UsWest1, sauceUsername, sauceAccessKey,
+          new CreateBuildOptions() { Name = "My Visual Build", Project = "csharp-project", Branch = "csharp-branch" });
+      VisualClient.CaptureDom = true;
```

Comment:
shouldn't we add a comment like 

// configure other options
VisualClient.CaptureDom = true